mockgcp/mockapikeys/apikeysv2.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2023 Google LLC
```

Comment:
Also this filename is apikeysv2.go, which seems to be different from the naming conventions you mentioned in the README: `Implement the core CRUD methods in <resourcename>.go.` or the [example mock resource PR](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/869/files#diff-2563d51fd13526bf66666826de6474615077cd2d9073b6f01aa340e24fff2897). Should the naming convention be adjusted or something? I'd lean towards an option that may help achieve consistency and enable the automation of this step in the future.

Reply:
Renaming to match the name of the implemented proto service.